comms/src/inbound_message_service/duplicate_msg_cache.rs

```diff
@@ -0,0 +1,117 @@
+//  Copyright 2019 The Tari Project
```

Comment:
This is a great feature add! I suggest calling everything here MessageCache rather than DuplicateMessageCache; because it sounds a bit like we're caching duplicate messages :)

Reply:
MessageCache sounds better, change applied